tests/publiccloud/prepare_tools.pm

```diff
@@ -82,10 +82,12 @@ sub run {
     assert_script_run("ipa --version");
 
     # Download and Install Terraform
-    my $terraform_url = get_var('TERRAFORM_URL', 'https://releases.hashicorp.com/terraform/0.11.10/terraform_0.11.10_linux_amd64.zip');
+    my $terraform_url = get_var('TERRAFORM_URL', 'https://releases.hashicorp.com/terraform/0.11.13/terraform_0.11.13_linux_amd64.zip');
```

Comment:
```suggestion
    my $terraform_version = '0.11.13';
    my $terraform_url = get_var('TERRAFORM_URL', 'https://releases.hashicorp.com/terraform/' . $terraform_version . '/terraform_' . $terraform_version . '_linux_amd64.zip');
```

Reply:
we could also have terraform version as job variable, as it might change often...

Reply:
I think we have TERRAFORM_URL is enough, as it contains that version...

Reply:
And if we really need something like this, you could build it in your test-suite with:
`TERRAFORM_URL=http://asldfjalsdfj%TERRAFORM_VERSION%asldfjlsadfj`